lib/rules/selector-not-notation/index.js

```diff
@@ -152,7 +165,9 @@ function fixSimple(not) {
 	not.nodes.push(firstSelector);
 
 	for (const s of simpleSelectors) {
-		const last = not.parent.last;
+		const last = getLastConsecutiveNot(not.parent.nodes);
+
+		assert(last);
```

Comment:
[suggest] This change will be no longer needed if the suggestion above is applied:

```suggestion
```